package.json

```diff
@@ -45,16 +45,16 @@
     "@fullhuman/postcss-purgecss": "3.0.0",
     "@sinonjs/fake-timers": "6.0.1",
     "@storybook/addon-a11y": "6.1.10",
-    "@storybook/addon-actions": "6.0.28",
+    "@storybook/addon-actions": "6.1.11",
```

Comment:
Could you please update all `@storybook/*` to 6.1.11. I'm not sure why they are all out of sync. We should have only one version.

Reply:
Done